bucket/mtpcopy.json

```diff
@@ -0,0 +1,9 @@
+{
+    "homepage": "https://github.com/kzmi/mtpcopy",
+    "description": "A Windows CLI tool for copying files from/to a portable device via MTP.",
+    "license": "GNU",
+    "version": "0.3.0",
+    "url": "https://github.com/kzmi/mtpcopy/releases/download/0.3.0/mtpcopy.exe",
+    "bin": "mtpcopy.exe",
+    "hash": "25768b655afcb8ef1ec9afb5d775c6b73ca44f9b8d35297a72cc94b4638910f8"
+}
```

Comment:
```suggestion
{
    "version": "0.3.0",
    "description": "A Windows CLI tool for copying files from/to a portable device via MTP",
    "homepage": "https://github.com/kzmi/mtpcopy",
    "license": "MIT",
    "architecture": {
        "64bit": {
            "url": "https://github.com/kzmi/mtpcopy/releases/download/0.3.0/mtpcopy.exe",
            "hash": "25768b655afcb8ef1ec9afb5d775c6b73ca44f9b8d35297a72cc94b4638910f8"
        }
    },
    "bin": "mtpcopy.exe",
    "checkver": "github",
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/kzmi/mtpcopy/releases/download/$version/mtpcopy.exe"
            }
        }
    }
}
```

Reply:
I've added your code in the file , anything else i can do that will make it better?

Reply:
> I've added your code in the file , anything else i can do that will make it better?

Nothing else. I think we have all the items needed.